frontend/public/components/edit-yaml.jsx

```diff
@@ -890,16 +890,15 @@ const EditYAML_ = connect(stateToProps)(
 );
 
 export const EditYAML = connectToFlags(FLAGS.CONSOLE_YAML_SAMPLE)(({ flags, ...props }) => {
-  const resources =
-    flags[FLAGS.CONSOLE_YAML_SAMPLE] && props.create
```

Comment:
The resource sidebar checks for create mode, so we can always fetch samples to get the snippets here.

https://github.com/openshift/console/blob/master/frontend/public/components/sidebars/resource-sidebar.jsx#L87

Reply:
You posted the answer just as I was typing this question :D